### ISSUE TITLE
Confirm that all the header data from units is being included

### ISSUE THREAD

**Issue by zross** [zross/sensorDataImport#29]
With the MicroAeth I was not recording the header detail on unit ID etc.

**Comment by zross** [comment]
In the GPS we are not extracting the "name" of the track. Is this important. In the example I have it's "Morning Ride"

**Comment by dwjack** [comment]
no, the name of the track (e.g., "morning ride") is not important.  

all the relevant metadata information is in the file name.

**Comment by zross** [comment]
For micropem and microaeth we record header information.

For hexoskin does not appear there is any header info

For abp, I can't tell if the "stray" info in header is needed.

**Comment by zross** [comment]
@dwjack OK on GPS, correct me if I'm wrong about hex and let me know if we need anything from ABP. I'm reading the abp file itself.
